src/types/member/generic.rs

```diff
@@ -22,6 +22,8 @@ pub struct GenericMember<C: CollectiveTrait> {
 	pub address: String,
 	/// GitHub handle of the member.
 	pub github: String,
+	/// Matrix chat handle.
+	pub matrix: Option<String>,
```

Comment:
@wirednkod is is marked as Optional here to not make the CI on past requests fail, but it should be required in the front-end.